test/e2e/common.go

```diff
@@ -39,6 +39,7 @@ func DefaultGardenConfig(projectNamespace string) *framework.GardenerConfig {
 
 // DefaultShoot returns a Shoot object with default values for the e2e tests.
 func DefaultShoot(name string) *gardencorev1beta1.Shoot {
+	purpose := gardencorev1beta1.ShootPurposeDevelopment
```

Comment:
Why do we need purpose development for the shoot?
Currently, the e2e shoots use purpose evaluation (default), but the event logging feature shouldn't depend on it.

Reply:
I didn't know that the default is 'evaluation'. Thanks.